### ISSUE TITLE
SequenceAnnotation doesn't export the strand

### ISSUE THREAD

**Issue by neilswainston** [SynBioDex/libSBOLc#16]
Hi there,

I know that I should be using SBOL 2, but I'm still on 1.1.

There is a problem that SequenceAnnotation doesn't export the strand. I've updated code locally, but would you like a fix?

Cheers,

Neil.

**Comment by bbartley** [comment]
Thanks for the fix, Neil.  Glad you are finding the library useful.

Documentation for the v2 C++ and Python libraries (both in beta) will be forthcoming in April. 

Bryan

**Comment by neilswainston** [comment]
Hi Bryan,

Thanks very much for the rapid response!

I’ve also made a pull request in pySBOL. I noticed that the binaries were shipped in pySBOL, so I updated the Mac OSX one (Windows presumably still needs doing).

And yes, I do find the library very useful. Thanks for all of your work on this.

Cheers,

Neil.

Neil Swainston, PhD
Senior Experimental Officer

Centre for Synthetic Biology of Fine and Speciality Chemicals (SYNBIOCHEM)
Manchester Institute of Biotechnology
University of Manchester
Manchester M1 7DN
United Kingdom





On 22 Mar 2016, at 20:42, bbartley <notifications@github.com<mailto:notifications@github.com>> wrote:


Thanks for the fix, Neil. Glad you are finding the library useful.

Documentation for the v2 C++ and Python libraries (both in beta) will be forthcoming in April.

Bryan
